### PR TITLE
doc: document how to apply a token with github-nodejs-bot

### PR DESCRIPTION
**Pull request by legendecas**
Document how to apply a fine-grained token with github-nodejs-bot's name.

After this landed, github-nodejs-bot tokens that are not fine-grained tokens like https://github.com/nodejs/docker-node/blob/main/.github/workflows/automatic-updates.yml#L28 may be updated to be fine-grained tokens.

### CODE REVIEW

File: APPLY_ACCESS_TOKEN.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Apply for a nodejs-github-bot token
+
+Automation in https://github.com/nodejs may require github access token to access
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:3]:
```suggestion
Automation in the `nodejs` GitHub Org may require access tokens to access
```

File: APPLY_ACCESS_TOKEN.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+permission scoped endpoints. In the case of such requirement, the access token can
+be requested to be created under the name of https://github.com/nodejs-github-bot.
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:5]:
```suggestion
be requested to be created under the name of [`@nodejs-github-bot`][].
```

File: APPLY_ACCESS_TOKEN.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Apply for a nodejs-github-bot token
+
+Automation in https://github.com/nodejs may require github access token to access
+permission scoped endpoints. In the case of such requirement, the access token can
+be requested to be created under the name of https://github.com/nodejs-github-bot. 
+
+Only fine-grained access token can be created under the name of https://github.com/nodejs-github-bot.
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:7]:
```suggestion
Creating classic tokens for [`@nodejs-github-bot`][] is not permitted, only fine-grained tokens are allowed.
```

File: APPLY_ACCESS_TOKEN.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+To create a fine-grained access token with nodejs-github-bot, submit a PR to
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:9]:
```suggestion
To create a fine-grained access token for [`@nodejs-github-bot`][], submit a PR to
```

File: APPLY_ACCESS_TOKEN.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+add the requested repo in the registry below, and a TSC member or a build WG member
+(who has access to the https://github.com/nodejs-github-bot account) needs to take
+action to create the fine-grained token and save the token to the requested repo's
+repository secrets.
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:13]:
```suggestion
action to create the fine-grained token and save the token as a repository secret.
```

File: APPLY_ACCESS_TOKEN.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+## Registry
+
+The "repo" is a string of the GitHub `<owner>/<repo>`. Generally, the token should
+only be created for repo in the https://github.com/nodejs organization.
+
+The "token name" is a string that the token can be referenced in the GitHub Action
+scripts. Like a token name of `RELEASE_PLEASE_TOKEN` can be accessed from the script
+as `${{ secrets.RELEASE_PLEASE_TOKEN }}`.
+
+Repo                        | Token name
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:27]:
```suggestion
The "Secret name" is a string that the token can be referenced in the GitHub Action
scripts. Like a token name of `RELEASE_PLEASE_TOKEN` can be accessed from the script
as `${{ secrets.RELEASE_PLEASE_TOKEN }}`.

Repo                        | Secret name
```

File: APPLY_ACCESS_TOKEN.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+Automation in https://github.com/nodejs may require github access token to access
+permission scoped endpoints. In the case of such requirement, the access token can
+be requested to be created under the name of https://github.com/nodejs-github-bot. 
+
+Only fine-grained access token can be created under the name of https://github.com/nodejs-github-bot.
+
+To create a fine-grained access token with nodejs-github-bot, submit a PR to
+add the requested repo in the registry below, and a TSC member or a build WG member
+(who has access to the https://github.com/nodejs-github-bot account) needs to take
+action to create the fine-grained token and save the token to the requested repo's
```
Comment (aduh95) [inline APPLY_ACCESS_TOKEN.md:12]: We should document the precise process for how to land such PR, create the secret, add it as a repo secret.

File: APPLY_ACCESS_TOKEN.md (hunk 1 above)
Comment (legendecas) [reply APPLY_ACCESS_TOKEN.md]: Thanks, updated!

**Comment by richardlau** [comment]
I don't think this document belongs here -- this repository is for the code running the bot server and not for the bot user.
I think https://github.com/nodejs/admin/ would be a better place for this.

**Comment by legendecas** [comment]
Well, this repo has the slogan of "@nodejs-github-bot's heart and soul" so I thought it would be easier to navigate here.

**Comment by legendecas** [comment]
Make sense. Closing in favor of https://github.com/nodejs/admin/pull/894.

**Pull request closed without merging**